k9mail/src/main/java/com/fsck/k9/view/K9WebViewClient.java

```diff
@@ -84,6 +85,8 @@ private Intent createBrowserViewIntent(Uri uri, Context context) {
 
     protected abstract void addActivityFlags(Intent intent);
 
+    protected abstract void addCacheControlHeader(WebResourceResponse response);
```

Comment:
maybe indicate that this might fail with an `IfAvailable` suffix?

Reply:
I don't know. Feels like an implementation detail, not like it's an inherent part of the API.